.github/workflows/try-state.yml

```
@@ -0,0 +1,100 @@
+# Runs `try-state` on all live runtimes in `polkadot-sdk`
+# TODO: Post to a Matrix channel if any fails.
+
+name: Check try-state
+on:
+  push:
+    branches:
+      - liam-try-state-scheduled-checks
+  schedule:
+    - cron: "0 * * * *"
+
+jobs:
+  check_try_state:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
```

```suggestion
      - name: Rust Cache
        uses: Swatinem/rust-cache@v2.7.0
        with:
          cache-on-failure: true
```
We could try use this cache action here if the cache is less than 10 GB.

---

I wonder if we can just use the build artifacts from the latest master commit. Re-building that over and over again is kind of a waste.

---

> I wonder if we can just use the build artifacts from the latest master commit. Re-building that over and over again is kind of a waste.

Yes this would be ideal. I didn't know we saved the artefacts - will look into using those thanks.